plugins/database/mysql/mysql.go

```diff
@@ -294,11 +293,13 @@ func (m *MySQL) RotateRootCredentials(ctx context.Context, statements []string)
 				continue
 			}
 
-			m := map[string]string{
-				"username": m.Username,
-				"password": password,
-			}
-			if err := dbtxn.ExecuteTxQuery(ctx, tx, m, query); err != nil {
+			// This is not a prepared statement because not all commands are supported
+			// 1295: This command is not supported in the prepared statement protocol yet
+			// Reference https://mariadb.com/kb/en/mariadb/prepare-statement/
+			query = strings.Replace(query, "{{username}}", m.Username, -1)
+			query = strings.Replace(query, "{{password}}", password, -1)
```

Comment:
Question . . . is there any way this could be leveraged for a SQL injection attack? If so, should we add guards like checking for special characters or something?

Reply:
Okay! I think that should be fine because the username is operator-provided and the password probably is originally too, and then is generated. Thanks!